azure-pipelines-nightly.yml

```diff
@@ -14,55 +14,103 @@ schedules:
-      ProductBinPath4: '$(Build.SourcesDirectory)\src\Microsoft.OData.Client\bin\$(BuildConfiguration)'
-      mainDll4: 'Microsoft.OData.Client.dll'
-      ProductBinPath5: '$(Build.SourcesDirectory)\src\CodeGen\bin\$(BuildConfiguration)'
-      mainDll5: 'Microsoft.OData.Service.Design.T4.dll'
-      ProductBinPath: $(Build.SourcesDirectory)\bin\$(BuildPlatform)\$(BuildConfiguration)\Product
-      NugetArtifactsDir: 'Nuget-Debug'
-  steps:
-    - template: nightly.yml
+variables:
+- name: __NugetSecurityAnalysisWarningLevel__
+  value: none
+- name: BuildConfiguration
+  value: release
+- name: BuildPlatform
+  value: anycpu
+- name: ProductBinPath
+  value: $(Build.SourcesDirectory)\bin\$(BuildPlatform)\$(BuildConfiguration)
+- name: runCodesignValidationInjection
+  value: false
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: MSSecurity-1ES-Build-Agents-Pool
+      image: MSSecurity-1ES-Windows-2019
+      os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling
+    stages:
+    - stage: build
+      jobs:
+      - job:  Debug	
+        timeoutInMinutes: 120
+        variables:	
+            buildPlatform: 'Any CPU'	
+            buildConfiguration: 'Debug'	
+            skipComponentGovernanceDetection: true	
+            snExe: 'C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\sn.exe'	
+            snExe64: 'C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\sn.exe'	
+            gacUtil: '$(Build.SourcesDirectory)\test\CommonAssemblies\Microsoft.VisualStudio.QualityTools.UnitTestFramework.dll'	
+            ProductBinPath1: '$(Build.SourcesDirectory)\src\Microsoft.OData.Edm\bin\$(BuildConfiguration)'	
+            mainDll1: 'Microsoft.OData.Edm.dll'	
+            ProductBinPath2: '$(Build.SourcesDirectory)\src\Microsoft.OData.Core\bin\$(BuildConfiguration)'	
+            mainDll2: 'Microsoft.OData.Core.dll'	
+            ProductBinPath3: '$(Build.SourcesDirectory)\src\Microsoft.Spatial\bin\$(BuildConfiguration)'	
+            mainDll3: 'Microsoft.Spatial.dll'	
+            ProductBinPath4: '$(Build.SourcesDirectory)\src\Microsoft.OData.Client\bin\$(BuildConfiguration)'	
+            mainDll4: 'Microsoft.OData.Client.dll'	
+            ProductBinPath5: '$(Build.SourcesDirectory)\src\CodeGen\bin\$(BuildConfiguration)'	
+            mainDll5: 'Microsoft.OData.Service.Design.T4.dll'	
+            ProductBinPath: $(Build.SourcesDirectory)\bin\$(BuildPlatform)\$(BuildConfiguration)\Product	
+            NugetArtifactsDir: 'Nuget-Debug'	
+        templateContext:	
+          outputs:	
+          - output: pipelineArtifact	
+            displayName: 'Publish Artifact - NuGet Debug Packages'	
+            targetPath: '$(Build.ArtifactStagingDirectory)\NuGet-Debug'	
+            artifactName: NuGetDebug	
+        steps:	
+          - template: /nightly.yml@self
+      - job:  Release
```

Comment:
The requests I raised in the `Debug` job section are relevant here as well:

- https://github.com/OData/odata.net/pull/2965/files#r1607127969
- https://github.com/OData/odata.net/pull/2965/files#r1607130100
- https://github.com/OData/odata.net/pull/2965/files#r1607132058
- https://github.com/OData/odata.net/pull/2965/files#r1607133198

Reply:
I've applied the changes here as well.